### PR TITLE
add utm medium to vast url

### PR DESCRIPTION
**Pull request by brycedorn**

### CODE REVIEW

File: elements/bulbs-video/components/revealed.test.js [hunk 1]
```diff
@@ -596,6 +596,7 @@ describe('<bulbs-video> <Revealed>', () => {
         utmTestData = {
           utmSource: 'test source',
           utmCampaign: 'test campaign',
+          utmMedium: 'test medium',
```
Comment (mparent61) [inline elements/bulbs-video/components/revealed.test.js]: search this file for `utm_campaign`, I see a few places where you probably want to add similar checks for `utm_medium` too.

**Comment by mparent61** [comment]
I'm looking into failing Browerstack / Travis CI tests.

**Comment by mparent61** [comment]
@brycedorn If tests pass for you locally, and you could create new `bulbs-elements` release, I can handle testing + rolling out to sites later today.

**Comment by jmelvnsn** [comment]
lgtm

**Comment by mparent61** [comment]
Tests pass locally, Travis broken due to BrowserStack account issues

**Pull request merged**